backend/controllers/fieldController.js

```diff
@@ -0,0 +1,126 @@
+const Field = require('../models/FieldSchema');
+const { validationResult } = require('express-validator');
+const asyncHandler = require('express-async-handler');
+const Admin = require("../models/adminSchema");
+
+// @desc Get all fields
+// @route GET /api/fields
+// @access Private -> Admin only
+const getFields = asyncHandler(async (req, res) => {
+  const admin = await Admin.findById(req.admin.id);
+  if (!admin) {
+    res.status(401);
+    throw new Error("Admin not found");
+  }
+  res.send(await Field.find({}));
```

Comment:
```suggestion
res.send(await Field.findOne({}));
```
I think this is fine because we only have one instance of Field but .find() will in theory turn a list so just to make sure we have the correct data type